src/nvim/getchar.c

```diff
@@ -2757,6 +2757,27 @@ int str_to_mapargs(const char_u *strargs, bool is_unmap, MapArguments *mapargs)
       parsed_args.unique = true;
       continue;
     }
+
+    if (STRNCMP(to_parse, "<desc=", 6) == 0) {
+      const char_u *p = to_parse + 6;
+      garray_T ga = GA_INIT(sizeof(char), 20);
+      while (*p != '>' && *p != NUL) {
+        if (*p == '\\') {
+          p++;  // skip escape char
+        }
+        ga_append(&ga, (char) *p);
```

Comment:
```suggestion
        ga_append(&ga, (char)(*p));
```

Reply:
You dropped this in a rebase.

Reply:
Oops . I forgot to pull before rebase :P

Reply:
added.